examples/leaflet/app/root.tsx

```diff
@@ -0,0 +1,54 @@
+import type { LinksFunction, MetaFunction } from "@remix-run/node";
+import {
+  Links,
+  LiveReload,
+  Meta,
+  Outlet,
+  Scripts,
+  ScrollRestoration,
+} from "@remix-run/react";
+import { ClientOnly } from "./client-only";
+import { Map } from "./map.client";
+
+export const links: LinksFunction = () => [
+  {
+    rel: "stylesheet",
+    href: "https://unpkg.com/leaflet@1.8.0/dist/leaflet.css",
+  },
+];
+
+export const meta: MetaFunction = () => ({
+  charset: "utf-8",
+  title: "New Remix App",
+  viewport: "width=device-width,initial-scale=1",
+});
+
+export default function App() {
+  const mapHeight = "400px";
+
+  return (
+    <html lang="en">
+      <head>
+        <Meta />
+        <Links />
+      </head>
+      <body>
+        <Outlet />
+        <ScrollRestoration />
+        <Scripts />
+        <LiveReload />
+
+        <ClientOnly
+          fallback={
+            <div
+              id="skeleton"
+              style={{ height: mapHeight, background: "#d1d1d1" }}
+            />
+          }
+        >
+          {() => <Map height={mapHeight} />}
+        </ClientOnly>
+      </body>
+    </html>
+  );
+}
```

Comment:
Can you please move the Map handling & display to a `app/routes/index.tsx` file?
People are used to at least having the index route, and I think having the whole app in `root.tsx` will confuse them.